### ISSUE TITLE
display "post a message" closer to the other messages

### ISSUE THREAD

**Issue by jum-s** [inventaire/inventaire-client#113]
within the trastion page, it is counter intuitive to have a message field to reply to someone far from the written messages.
Would be more natural to have the field box right after the messages, then the "next action" possibility
![selection_010](https://user-images.githubusercontent.com/5363918/30109983-bc8a2d34-9308-11e7-89ba-8f328e054427.png)

**Comment by maxlath** [comment]
the motivation here was to have the actions and their legend made very visible: as it's a discussion, you expect to find a "post a message" box or something of this kind, but all the actions are new to most users. It tries to borrow heavily from Github issues/PR flows, the closest thing I could think of that merges comments and actions:
![pr](https://user-images.githubusercontent.com/1596934/30113002-81d88784-9313-11e7-958b-3d980ae02793.png)

**Comment by jum-s** [comment]
size is not the same, on gh the field box is twice as big as the action in between. it encourages more to answer than click "delete branch"
on a social network, the focus should be to engage a dialogue with someone, the actions are the least of your concern, i wanted to reply you first, then lend the book
borrowing heavily from github could be a "reply" button and next to it a "reply and [action]" button

**Comment by maxlath** [comment]
yes, having the action buttons near the "Send" button could do, but the actions should really be visible - as it's what keeps inventories in sync with transactions* - and need space to be explain properly as it's very specific to this social network

(*): actions allow
      - shared books to be marked as busy until the owner clicks the "confirm return" button
      - given or sold books to move from one inventory to the other once the receiver confirmed reception
